torchgeo/datamodules/substation.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Substation datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+import torch
+from torch.utils.data import random_split
+
+from ..datasets import Substation
+from .geo import NonGeoDataModule
+
+
+class SubstationDataModule(NonGeoDataModule):
+    """Substation Data Module with train-test split and transformations.
+
+    .. versionadded:: 0.7
+    """
+
+    def __init__(
+        self,
+        root: str,
+        batch_size: int = 64,
+        num_workers: int = 0,
+        val_split_pct: float = 0.2,
+        test_split_pct: float = 0.2,
+        bands: list[int] = [1, 2, 3],
+        num_of_timepoints: int = 4,
+        timepoint_aggregation: str = 'median',
```

Comment:
Any of these parameters that just get passed straight to the dataset can be removed, they are already covered in `kwargs`.